lib/src/changes.dart

```diff
@@ -207,6 +207,145 @@ final Map<String, String> _textReplacements = {
   // dart:io/string_transformer.dart
   'SYSTEM_ENCODING': 'systemEncoding',
 
+  // dart:_http/http.dart
+  'HttpStatus.CONTINUE': 'HttpStatus.continue_',
```

Comment:
Does this constant really end with an underscore?

Reply:
Seems so. Reserved words are annoying.